### PR TITLE
SRCH-3047 update the ASIS README to use search-services

### PR DESCRIPTION
**Pull request by krbhavith**
## Summary
SRCH-3047 update the ASIS README to use search-services
- Updates to README
- delete docker-compose.yml
- gem version bumps
- using 'ES_HOSTS' as env variable.
 
### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review:
 
#### Functionality Checks
 
- [x] Code is functional, as tested locally.

- [x] Tests have been added or updated to cover the proposed changes. If not, please explain why tests aren't needed:
 
- [x] Automated checks pass, if applicable. If CodeClimate checks do not pass, explain reason for failures:
 
- [x] If your changes will be tested manually, you have run `bundle update` and committed your changes to Gemfile.lock.

- [x] You have merged the latest changes from the target branch (usually `main`) into your branch.
 
- [x] Your target branch is `main` or `production`, or you have specified the reason for an alternate branch here:
 
- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.
 
- [x] You have squashed your commits into a single commit (exceptions: your PR includes commits with formatting-only changes, such as required by Rubocop or Cookstyle, or if this is a feature branch that includes multiple commits)
 
- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket
 
#### Process Checks

- [x] You have specified an "Assignee", and if necessary, additional reviewers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,55 +33,21 @@ We use bundler to manage gems. You can install bundler and other required gems l
     gem install bundler
     bundle install
 
-### Elasticsearch
+### Docker Services
 
-We're using [Elasticsearch](http://www.elasticsearch.org/) (>= 6.8) for fulltext search.
+The required services (MySQL, Elasticsearch, etc.) can be run using Docker. Please refer to [searchgov-services](https://github.com/GSA/search-services) for detailed instructions on centralized configuration for services.
```
Comment (MothOnMars) [inline README.md:38]: For consistency with `bot_detection`'s README and for simplicity, let's change this to:
```
### Services

The required services (MySQL, Elasticsearch, & Redis) can be run using Docker. Refer to [search-services](https://github.com/GSA/search-services) for detailed instructions.
```

File: README.md [hunk 2]
```diff
@@ -33,55 +33,21 @@ We use bundler to manage gems. You can install bundler and other required gems l
...
 ### Redis
 
-Sidekiq (see below) uses [Redis](http://redis.io), so make sure you have that installed and running. You can install it it from the [Redis website](https://redis.io/download), or run it using Docker:
-
-    docker-compose up redis
-
-To run Elasticsearch, Kibana, and Redis, you can simply run:
-
-    docker-compose up
+Sidekiq (see below) uses [Redis](http://redis.io), so make sure you have that installed and running.
```
Comment (MothOnMars) [inline README.md:50]: We can remove this section on Redis, as we're noting above that it is a required service, run by search-services.

**Review by MothOnMars — DISMISSED** [review]
A few more tweaks. After talking w/ Dan re. the I14Y README, we agreed that we should keep the language re. the services in each repo to the bare minimum, to avoid duplicating any info that's covered in the search-services README.

**Comment by lorawoodford** [comment]
Paging @danswick here too since I don't know where we stand on this element of the dev approach:

`simplify the instructions on specifying the ES_HOSTS`

Was the decision made to call out the env vars in each repo's readme (since they are differently named in each repo)?   Looks like this open PR for search-gov does call out the ES_HOSTS var in the searchgov repo's readme:  https://github.com/GSA/search-gov/pull/942/files This same language could be copy/pasted into ASIS if we're going that route.

Or, have we decided to demonstrate how to set various env vars only once in the search services repo (I checked that repo and the open PR and didn't see this)?  

Generally, this LGTM, I just want to make sure that this documentation step as requested in JIRA isn't overlooked/bypassed.

File: README.md [hunk 3]
```diff
@@ -33,55 +33,9 @@ We use bundler to manage gems. You can install bundler and other required gems l
...
-### Redis
-
-Sidekiq (see below) uses [Redis](http://redis.io), so make sure you have that installed and running. You can install it it from the [Redis website](https://redis.io/download), or run it using Docker:
-
-    docker-compose up redis
-
-To run Elasticsearch, Kibana, and Redis, you can simply run:
-
-    docker-compose up
+The required services (MySQL, Elasticsearch, & Redis) can be run using Docker. Refer to [search-services](https://github.com/GSA/search-services) for detailed instructions.
```
Comment (lorawoodford) [inline README.md:38]:
```suggestion
The required services (MySQL, Elasticsearch, & Redis) can be run using Docker. Refer to [search-services](https://github.com/GSA/search-services) for detailed instructions.

The Elasticsearch services provided by `searchgov-services` is configured to run on the default port, [9200](http://localhost:9200/). To use a different host (with or without port) or set of hosts, set the `ES_HOSTS` environment variable. For example, use following command to run the specs using Elasticsearch running on `localhost:9207`:

    ES_HOSTS=localhost:9207 bin/rspec spec
```

**Review by lorawoodford — CHANGES_REQUESTED** [review]
Alright, after meeting with @danswick we'll proceed in calling out the ES_HOSTS env var in each repo's readme.  Just add the suggested paragraph, then this will be good to go.  Thanks for your patience!

**Review by lorawoodford — APPROVED** [review]
LGTM

File: README.md [hunk 4]
```diff
@@ -33,55 +33,13 @@ We use bundler to manage gems. You can install bundler and other required gems l
...
-### Redis
-
-Sidekiq (see below) uses [Redis](http://redis.io), so make sure you have that installed and running. You can install it it from the [Redis website](https://redis.io/download), or run it using Docker:
-
-    docker-compose up redis
-
-To run Elasticsearch, Kibana, and Redis, you can simply run:
-
-    docker-compose up
+    ES_HOSTS=localhost:9207 bin/rspec spec
```
Comment (MothOnMars) [inline README.md:42]: Per my comment in the ticket, this should be:
```
ES_HOSTS=localhost:9207 bundle exec rspec spec
```

**Review by MothOnMars — CHANGES_REQUESTED** [review]

**Review by MothOnMars — APPROVED** [review]
LGTM

**Pull request merged**